tests/Rector/Architecture/Factory/NewObjectToFactoryCreateRector/config.yml

```diff
@@ -0,0 +1,5 @@
+services:
+    Rector\Rector\Architecture\Factory\NewObjectToFactoryCreateRector:
+        Rector\Tests\Rector\Architecture\Factory\NewObjectToFactoryCreateRector\Source\MyClass:
+            - Rector\Tests\Rector\Architecture\Factory\NewObjectToFactoryCreateRector\Source\MyClassFactory
+            - create
```

Comment:
What about adding `class` and `method` keys, so it's more obvious this is not n-list?

```diff
 services:
     Rector\Rector\Architecture\Factory\NewObjectToFactoryCreateRector:
         Rector\Tests\Rector\Architecture\Factory\NewObjectToFactoryCreateRector\Source\MyClass:
-            - Rector\Tests\Rector\Architecture\Factory\NewObjectToFactoryCreateRector\Source\MyClassFactory
-            - create
+            class: Rector\Tests\Rector\Architecture\Factory\NewObjectToFactoryCreateRector\Source\MyClassFactory
+            method create
```